demos/mqtt/mqtt_demo_mutual_auth/mqtt_demo_mutual_auth.c

```diff
@@ -1561,7 +1565,8 @@ int main( int argc,
             else
             {
                 /* If TLS session is established, execute Subscribe/Publish loop. */
-                returnStatus = subscribePublishLoop( &mqttContext, &clientSessionPresent );
+                mqttSessionPresent = true;
```

Comment:
If this is always true why have it?

Reply:
Initially it is false, I have initialed it to false and once The MQTT connection is made it will be set to true.